web/ui/react-app/src/pages/serviceDiscovery/Services.tsx

```diff
@@ -10,10 +10,12 @@ import { API_PATH } from '../../constants/constants';
 import { KVSearch } from '@nexucis/kvsearch';
 import { Container } from 'reactstrap';
 import SearchBar from '../../components/SearchBar';
+import internal from 'stream';
```

Comment:
Did this line get added by accident? Seems like it shouldn't be there.

Reply:
Removed.